kontrollsamtale/domain/src/main/kotlin/no/nav/su/se/bakover/kontrollsamtale/domain/opprett/SakOpprettKontrollsamtale.kt

```diff
@@ -0,0 +1,33 @@
+package no.nav.su.se.bakover.kontrollsamtale.domain.opprett
+
+import arrow.core.Either
+import arrow.core.left
+import no.nav.su.se.bakover.common.domain.tid.periode.NonEmptyIkkeOverlappendePerioder
+import no.nav.su.se.bakover.common.tid.periode.toMåned
+import no.nav.su.se.bakover.domain.Sak
+import no.nav.su.se.bakover.kontrollsamtale.domain.Kontrollsamtale
+import no.nav.su.se.bakover.kontrollsamtale.domain.Kontrollsamtaler
+import no.nav.su.se.bakover.kontrollsamtale.domain.regnUtFristFraInnkallingsdato
+import java.time.Clock
+
+/**
+ * Oppretter en ny, planlagt kontrollsamtale.
+ * Frist - se [regnUtFristFraInnkallingsdato]
+ */
+fun Sak.opprettKontrollsamtale(
+    command: OpprettKontrollsamtaleCommand,
+    eksisterendeKontrollsamtalerForSak: Kontrollsamtaler,
+    clock: Clock,
+): Either<KanIkkeOppretteKontrollsamtale, Pair<Kontrollsamtale, Kontrollsamtaler>> {
+    if (this.søknadsbehandlinger.isEmpty()) return KanIkkeOppretteKontrollsamtale.IngenStønadsperiode.left()
+    val stønadsperioder = this.hentStønadsperioder() as NonEmptyIkkeOverlappendePerioder
```

Comment:
Hva hvis du har en søknadsbehandling som ikke er iverksatt? hentStønadsperioder() sjekker for søknadsbehandlingsvedtak. Får da en tom liste tilbake som vil kaste exception

Reply:
Good catch. Fikser.